examples/resources/genesyscloud_journey_outcome_predictor/resource.tf

```diff
@@ -0,0 +1,5 @@
+resource "genesyscloud_journey_outcome_predictor" "example_journey_outcome_predictor_resource" {
+  outcome {
+    id = data.genesyscloud_journey_outcome.exampleOutcome.id
+  }
+}
```

Comment:
You'll have to change this to `outcome_id = data.genesyscloud_journey_outcome.example_outcome.id` and rerun `go generate`